packages/react-native-renderer/src/ReactNativeBridgeEventPlugin.js

```diff
@@ -103,13 +141,27 @@ function accumulateDispatches(
 ): void {
   if (inst && event && event.dispatchConfig.registrationName) {
     const registrationName = event.dispatchConfig.registrationName;
-    const listener = getListener(inst, registrationName);
-    if (listener) {
+    // Since we "do not look for phased registration names", that
+    // should be the same as "bubbled" here, for all intents and purposes...?
+    const listeners = getListeners(
```

Comment:
Same comments about `getListeners` => `pushListeners` or something to prevent unnecessary allocation and later unnecessary `accumulateInto` checks.